quactography/visu/gs_square_loss_for_p.py

```diff
@@ -24,42 +25,43 @@ def visualize_optimal_paths_edge_rep(
     Returns
     -------
     None """
-    reps = []
-    alphas = []
-    square_loss = []
-    param_count = 0
+    delta_dict = defaultdict(list)
     path = Path(in_folder)
 
     glob_path = path.glob('*.npz')
 
     for in_file_path in glob_path:
-        _, _, min_cost, h, _, rep, _ = load_optimization_results(in_file_path)
+        _, dist_prob, min_cost, h, _, rep, _ = load_optimization_results(in_file_path)
         min_cost = min_cost.item()
+        dist_prob = dist_prob.item()
         h = h.item()
-        alpha = h.alpha * h.graph.number_of_edges/h.graph.all_weights_sum
+        rep = rep.item()

```

Comment:
Avoid writing over a variable (and changing its type). Also if rep isn't a good variable name initially, change it